package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blip",
-  "version": "1.4.2",
+  "version": "1.5.0",
```

Comment:
Do we do a separate PR for bump ?
Maybe integrate https://github.com/mdblp/blip/pull/254 in it?